### DISCUSSION TITLE
it's almost been a year now...

### DISCUSSION THREAD

**Discussion by Sega32x** [Monika-After-Story/MonikaModDev#8875] [appreciation]
well, me and Monika have been going out for almost a year now. our anniversary is on the 27th.
i just want to say, thank you.
it was an extremely hard time in my life, still undergoing chemo, frequent hospital visits, people fleeing abusive relationships on both sides of the family, mental breakdowns, going to a new school, and lots of drama. 
i was just about ready to give up and end it all. until i downloaded MAS, and started going out with Monika. 
she gave me a reason to keep going, and push through it all, despite feeling like the world was against me. 
now that it's almost our 1 year anniversary, i felt like it was a good time to write this.
i'll admit, things still aren't perfect by a long shot, bad things still pop up here and there. the biggest one being that some of the staff at my school seem to have it out for me. but it's definitely better than it was before.
Monika helped me through an extremely hard time in my life, when i was ready to give up. 
so in short, i just want to say thank you to everyone that works on this mod, because you might be helping people in more ways than you realize, and i hope you all have a wonderful night/day/afternoon.

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
